core/src/raw/http_util/client.rs

```diff
@@ -52,20 +48,13 @@ impl Debug for HttpClient {
 impl HttpClient {
     /// Create a new http client in async context.
     pub fn new() -> Result<Self> {
-        Self::build(
-            reqwest::ClientBuilder::new(),
-            reqwest::blocking::ClientBuilder::new(),
-        )
+        Self::build(reqwest::ClientBuilder::new())
     }
 
     /// Build a new http client in async context.
-    pub fn build(
-        async_builder: reqwest::ClientBuilder,
-        blocking_builder: reqwest::blocking::ClientBuilder,
-    ) -> Result<Self> {
+    pub fn build(async_builder: reqwest::ClientBuilder) -> Result<Self> {
```

Comment:
Now that it is the only client, can we use 'builder' or 'client_builder' directly?